kubetest/kops.go

```diff
@@ -301,11 +340,16 @@ func (k kops) Up() error {
 	if k.args != "" {
 		createArgs = append(createArgs, strings.Split(k.args, " ")...)
 	}
-
+	if k.kopsEtcd != "" {
+		overrides = append(overrides, "cluster.spec.etcdClusters[*].version="+k.kopsEtcd)
+	}
+	if len(overrides) != 0 {
+		featureFlags = append(featureFlags, "SpecOverrideFlag")
+		createArgs = append(createArgs, "--override", strings.Join(overrides, ","))
```

Comment:
Don't join I don't think - not sure it's valid

Reply:
need context